src/coord/rmr/command.h

```diff
@@ -33,8 +33,13 @@ typedef struct {
   /* Slots info offset - 0 if not set (first argument is always the command) */
   uint32_t slotsInfoArgIndex;
 
-  /* if not NULL, this value indicate to which shard the command should be sent */
-  char *targetShard;
+  /* if not NULL, this value indicate to which shard the command should be sent.*/
+  const char *targetShard;
```

Comment:
Making it `const` yields warnings when freeing it. Consider not having it const, or fix the warning in the `MRCommand` free function